### ISSUE TITLE
Add JSON Schema files

### ISSUE THREAD

**Issue by orta** [zeit/schemas#50]
:wave: - One thing I used for Peril that has been really useful is to [export a JSON schema](https://github.com/danger/peril/blob/master/peril-settings-json.schema) in the repo for the app settings file. This is auto-generated from the objects in the repo.

Then you can use this arbitrarily in any JSON file by using `"$schema": "[url]"`:

<img width="1136" alt="screen shot 2019-02-18 at 6 22 10 am" src="https://user-images.githubusercontent.com/49038/52947797-c4dba600-3345-11e9-9719-7220314f606e.png">

Now I've got validations + docs inline in my editor.

**Comment by simonhaenisch** [comment]
Yup would be nice if you compile your schemas into json in like a `dist` folder or sth, so we could use them as

```
https://unpkg.com/@zeit/schemas/dist/deployment/config
```
 
:nerd_face:

**Comment by dtinth** [comment]
For a workaround, I have submitted a pull request #59 with an action to automatically update JSON files from JS files. These JSON files are available in [my fork](https://github.com/dtinth/vercel-schemas/tree/master/json/deployment).

**Comment by dhruvin2910** [comment]
Any updates on this @anatrajkovska?

**Comment by GerryFerdinandus** [comment]
A problem request has been opened to add support  for vercel.json
https://github.com/SchemaStore/schemastore/issues/1942
How to continue with this issue?
 There is no json file here that can be link to.
